### PR TITLE
invano's branch

### PR DESCRIPTION
**Pull request by imbillow**
#65

maybe need rebase

cc @invano

### CODE REVIEW

File: scripts/run_ida_script.py [hunk 1]
```diff
@@ -16,6 +21,17 @@
 logger = logging.getLogger(__name__)
 
 
+def script_path_to_args(script_path):
```
Comment (williballenthin) [inline scripts/run_ida_script.py:24]: can we use shlex for this?

https://docs.python.org/2.7/library/shlex.html#shlex.split

**Comment by XVilka** [comment]
@imbillow can you please rebase against the current master? Since some of your changes were merged already?

File: tests/test_idaapi.py [hunk 2]
```diff
@@ -584,10 +583,13 @@ def test_function_names(kernel32_idb, version, bitness, expected):
         else "__BaseDllInitialize@12"
     )
 
-    if version > 500:
-        with pytest.raises(KeyError):
-            # this is issue #7.
-            _ = api.idc.GetFunctionName(0x689018E5)
+    if version >= 720:
+        assert api.idc.GetFunctionName(0x689018E5) == "_InternalFindAtom@12"
+    # elif version > 500:
```
Comment (XVilka) [inline tests/test_idaapi.py:588]: Can you try to remove this comment, since we now have the means to autotest this?

**Review by XVilka — CHANGES_REQUESTED** [review]

File: tests/test_idaapi.py (hunk 2 above)
Comment (XVilka) [reply tests/test_idaapi.py:588]: I meant, what is the name for 6.x databases?

File: scripts/run_ida_script.py (hunk 1 above)
Comment (imbillow) [reply scripts/run_ida_script.py]: can, it's nice

File: tests/test_idaapi.py (hunk 2 above)
Comment (imbillow) [reply tests/test_idaapi.py]: OK

Comment (imbillow) [reply tests/test_idaapi.py]: for 6.x idb, `api.idc.GetFunctionName(0x689018E5) == "sub_689017d4"`

looks doesn't make sense.

it should be `"sub_{:08X}".format(123)`

Comment (imbillow) [reply tests/test_idaapi.py]: now ignore this Error raise

image

So sometimes there are inconsistencies

**Pull request merged**